gameobjects/units/cUnitUtils.h

```diff
@@ -10,23 +10,15 @@
 
   */
 
-#ifndef UNIT_UTILS
-#define UNIT_UTILS
-class cUnitUtils {
-
-private:
-	static cUnitUtils *instance;
-
-protected:
-	cUnitUtils();
+#pragma once
 
-public:
-
-	static cUnitUtils *getInstance();
+class cUnitUtils {
+  protected:
+  	cUnitUtils();
 
-	int findUnit(int type, int iPlayerId); // find the first unit of type belonging to player Id.
-	int findUnit(int type, int iPlayerId, int iIgnoreUnitId);
+  public:
+	  static cUnitUtils& getInstance();
 
+	  int findUnit(int type, int iPlayerId); // find the first unit of type belonging to player Id.
```

Comment:
Is this even used? If not, we can delete it.

I believe some functions have been added to `player` for finding units and such (which are used)